content/2016.12.how-to-develop-a-webapp-with-Kinto.md

```diff
@@ -0,0 +1,247 @@
+Title: How to develop a WebApp with Kinto
+Date: 2016-12-01 10:20
+Slug: develop-a-chat-webapp-with-kinto
+Authors: Enguerran Colson, Frank Taillandier, Kinto Team
+Summary: An example of how Kinto handles a webapp's data storage.
+
+_This post was [originally published in french on Enguerran Colson's blog](https://blog.ticabri.com/2016/11/29/developper-une-webapp-avec-kinto/)._
+
+## Our goal
+
+Hi folks, today, we are gonna create a simple WebApp that allows visitors to chat in a room accessible from a public URL.
+
+![KintoChat, our instant chat WebApp](images/kintochat-fullscreen.png)
+{.image}
+
+* A really simple chat webapp with minimal UI.
```

Comment:
I propose to add this before the list

> What will we need?

→ https://github.com/DirtyF/kinto-website/commit/8876c4257996d18f7041eaa46c1717deb050ecb2